crates/pyhq/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pyhq"
-version = "0.12.0"
+version = "0.13.0"
```

Comment:
Maybe we should also move the version to a workspace property? I'm not sure why I haven't done that already, maybe there was some problem with it :thinking:

Reply:
I agree. I will do it in the next version